utilities_export.py

```
@@ -1,3 +1,547 @@
+        
+    def exportRootAfter(self, pagenow):
+        self.window.lift()
+        if(pagenow > self.endpageVar.get()):
+            self.window.withdraw()
+            self.count = 0
+            self.progresssearch['value'] = self.count
+            self.botaoaplicar.config(state='normal')
+            self.exporting = False
+            try:
+                self.docatual.close()
+            except:
```

Ruff

Do not use bare `except`

(E722)